### PR TITLE
added assetions to verify number of requests

### PR DESCRIPTION
**Pull request by msakowicz**

### CODE REVIEW

File: features/environment.py [hunk 1]
```diff
@@ -38,8 +38,9 @@ def before_scenario(context, scenario):
     context.session_id = context.executor.get_session_id()
     # scenario.name = '%s_%s' % (scenario.name, context.browser.upper())
 
-    if "apple_test" in scenario.tags and (context.browser not in "Safari"):
-        scenario.skip("SCENARIO SKIPPED as iOS system and Safari is required for ApplePay test")
+    if "apple_test" in scenario.tags and (context.browser not in "safari"):
+        if 'iP' not in CONFIGURATION.REMOTE_DEVICE:
```
Comment (james-simpson-ST) [inline features/environment.py]: Do we not want to run this on mac os? I thought it should work if they use a mac with a Touch ID sensor or Face ID, so we should test it on them too

Comment (msakowicz) [reply features/environment.py]: Yes, I've run set of full test on local chrome and smoke test on firefox on browserstack. At this moment one test is failing. This one which is related to update_jwt feature and it is probably connected with bug which exist on develop branch https://securetrading.atlassian.net/browse/STJS-576 (duplicated jsinit request). So we have to wait until this bug will be fixed and after that we can merge this branch to master.

Comment (msakowicz) [reply features/environment.py]: > Do we not want to run this on mac os? I thought it should work if they use a mac with a Touch ID sensor or Face ID, so we should test it on them too

Actually, this 'if' condition should run apple pay test on all mac devices with Safari and also on all mobile devices with ios system and skip it for the rest (windows, android, etc)
'if 'iP' not in CONFIGURATION.REMOTE_DEVICE '-> 'iP' is a shortcut for iPhone and iPad :)
I'll make double check and confirm that is working as expected :)

Comment (msakowicz) [reply features/environment.py]: @james-simpson-ST This branch works correctly together with current develop https://www.travis-ci.org/github/SecureTrading/js-payments/builds/666010941

I can also confirm that apple pay tests are launched on Safari/mac and ios mobile devices but skipped on other browsers and systems

**Pull request merged**